helmfile/charts/notify-api/templates/deployment.yaml

```diff
@@ -40,51 +40,43 @@ spec:
             [
               "sh",
               "-c",
-              "until nslookup {{ .Values.postgresHost }}; do echo waiting for postgres; sleep 2; done;",
+              "until nslookup $POSTGRES_HOST; do echo waiting for postgres; sleep 2; done;",
             ]
+          env:
+            - name: POSTGRES_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: notify-api
+                  key: POSTGRES_HOST   
         - name: migrate-db
-          image: api
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
we will have a longer reference for the image if I get this right?

Reply:
correct - but the image and tag were set in the kustomization.yaml anyway just pointing to the alias "api" - so it's no real difference.